render.yaml

```diff
@@ -0,0 +1,25 @@
+databases:
+  - name: venulet_pgsql
+    ipAllowList: []
+    databaseName: _VenuLet_production
+    user: _VenuLet
+    region: frankfurt
+
+services:
+  - type: web
+    name: venulet_rails
+    env: ruby
+    repo: https://github.com/fordburgess/event-hosting
+    branch: master
+    rootDir: .
+    healthCheckPath: /
+    buildCommand: "./bin/render-build.sh"
+    startCommand: "bundle exec puma -C config/puma.rb"
+    envVars:
+      - key: DATABASE_URL
+        fromDatabase:
+          name: venulet_pgsql
+          property: connectionString
+      - key: RAILS_MASTER_KEY
```

Comment:
in your render dashboard you need to add your rails master key to to the RAILS_MASTER_KEY environment var

you can find it in `config/master.key/ or `config/environments/production.key` if its already generated, if not run `rails credentials:edit -e production` and a new one will be created, use that when configuring the render services